protos/dcs/coalition/v0/coalition.proto

```diff
@@ -9,6 +9,16 @@ service CoalitionService {
   // https://wiki.hoggitworld.com/view/DCS_func_addGroup
   rpc AddGroup(AddGroupRequest) returns (AddGroupResponse) {}
 
+  // Focussed on statics (linked statics - see `AddLinkedStatic`)
+  // https://wiki.hoggitworld.com/view/DCS_func_addStaticObject
+  rpc AddStaticObject(AddStaticObjectRequest)
+      returns (AddStaticObjectResponse) {}
+
+  // Focussed on properties relevant to linked static objects
+  // https://wiki.hoggitworld.com/view/DCS_func_addStaticObject
+  rpc AddLinkedStatic(AddLinkedStaticRequest)
+      returns (AddLinkedStaticResponse) {}
+
```

Comment:
I'd personally prefer having only one `AddStaticObject` method and distinguish between linked and not linked with a `oneof` inside of the payload. But I could live with having the methods, too.